app/js/components/manageApps/AddonList.jsx

```diff
@@ -9,40 +9,41 @@
  */
 import React from 'react';
 
-export const AddonList = ({handleDownload, install, appList, openPage, openModal}) => {
+export const AddonList = ({handleDownload, appList, openPage, openModal}) => {
   return (
     <tbody>
       {
         appList.map((app, key) => {
           return (
             <tr key={key}>
-              <td onClick={() => openPage(app)}>
+              <td onClick={() => openPage(app.appDetails)}>
                 <img
-                  src={app.icons !== null && app.icons !== undefined ? 
-                    `/${location.href.split('/')[3]}/owa/openmrs-addonmanager/${app.icons[48]}` : 
+                  src={app.appDetails && app.appDetails.icon !== null && app.appDetails.icons !== undefined ? 
```

Comment:
Your line seem too long more than 80 chars

Reply:
Thanks @Ewanjiru  would refactor that 👍